src/main/kotlin/com/celuveat/member/adapter/out/oauth/config/SocialLoginApiClientConfig.kt

```diff
@@ -1,6 +1,8 @@
 package com.celuveat.member.adapter.out.oauth.config
```

Comment:
패키지도 social로 가는건 어떤가요?
```suggestion
package com.celuveat.member.adapter.out.social.config
```

Reply:
@TaeyeonRoyce 헉 이걸 못봤네용... 좋아여!!!
다음 feature 에서 반영하겠습둥